frontend/src/App.js

```diff
@@ -5,21 +5,16 @@ import Header from './Header';
 import Footer from './Footer';
 import Home from './MainPage/Home'
 import {Route, Switch} from "react-router-dom";
-
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
are you sure that its correct way of import this styles?

Reply:
![image](https://user-images.githubusercontent.com/38259449/97321605-1dd04100-1880-11eb-8430-f98b0356b916.png)

Reply:
😭